modules/openy_map_lb/src/Plugin/Block/LocationFinder.php

```diff
@@ -163,8 +163,14 @@ protected function getAmenities() {
       $result['type'] = 'groups';
     }
 
+    $lang_code = \Drupal::service('language_manager')->getCurrentLanguage()->getId();
```

Comment:
@froboy please use dependency injection for this service.

Reply:
@andrebonon updated.

Reply:
Thanks @froboy! Looks good!